gradle/publish.gradle

```
@@ -0,0 +1,78 @@
+apply plugin: 'com.novoda.build-properties'
+apply plugin: 'com.jfrog.bintray'
+apply plugin: 'maven-publish'
+
+buildProperties {
+    cli {
+        using(project)
+    }
+
+    bintray {
+        def bintrayCredentials = {
+            return isDryRun() ?
+                    ['bintrayOrg': 'n/a', 'bintrayRepo': 'n/a', 'bintrayUser': 'n/a', 'bintrayKey': 'n/a'] :
+                    new File("${System.getenv('BINTRAY_PROPERTIES')}")
+        }
+        using(bintrayCredentials()).or(cli)
+        description = '''This should contain the following properties:
+                        - bintrayOrg: name of the Bintray organisation to deploy the artifacts to
+                        - bintrayRepo: name of the repo of the organisation to deploy the artifacts to
+                        - bintrayUser: name of the account used to deploy the artifacts
+                        - bintrayKey: API key of the account used to deploy the artifacts'''
+                .stripIndent()
+    }
+}
+
+
+def pomConfig = {
+    licenses {
+        license {
+            name "The Apache Software License, Version 2.0"
+            url "http://www.apache.org/licenses/LICENSE-2.0.txt"
+            distribution "repo"
+        }
+    }
+    developers {
+        developer {
+            id "novoda"
+            name "Novoda GmbH"
+        }
+    }
+    scm {
+        url "https://github.com/novoda/aws-v4-signer-java"
+    }
+}
+
+bintray {
+
+    user = project.buildProperties.bintray['bintrayUser'].string
+    key = project.buildProperties.bintray['bintrayKey'].string
+
+    pkg {
+        userOrg = project.buildProperties.bintray['bintrayOrg'].string
+        repo = project.buildProperties.bintray['bintrayRepo'].string
+        name = "aws-v4-signer"
+        licenses = ['Apache-2.0']
+        vcsUrl = 'https://github.com/novoda/aws-v4-signer-java'
+    }
+}
+
+afterEvaluate {
```

I believe that instead using an `afterEvaluate` here we could do directly:
```suggestion
project.publishing.publications.all { publication ->
    ...
}
```